src/celldega/sp/__init__.py

```diff
@@ -0,0 +1,258 @@
+"""
+Module for spatial operation and analysis
+"""
+
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+import matplotlib.cm as cm
+import geopandas as gpd
+import pandas as pd
+
+def calc_distance_to_polygon(
```

Comment:
Maybe we can start calling this ROI instead of polygon in our functions?

Reply:
fixed.